cli/mender-artifact/write.go

```diff
@@ -333,6 +334,41 @@ func makeTypeInfo(ctx *cli.Context) (*artifact.TypeInfoV3, *artifact.TypeInfoV3,
 	return typeInfoV3, augmentTypeInfoV3, nil
 }
 
+func getSoftwareVersion(artifactName, softwareFilesystem, softwareName, softwareVersion string, noDefaultSoftwareVersion bool) map[string]string {
+	result := map[string]string{}
+	softwareVersionName := "rootfs-image"
+	if softwareFilesystem != "" {
+		softwareVersionName = softwareFilesystem
+	}
+	if softwareName != "" {
+		softwareVersionName += fmt.Sprintf(".%s", softwareName)
+	}
```

Comment:
For `write rootfs-image`, the default for `softwareVersionName` is `""`, but for `write module-image` it is the name of the payload type (`-T` parameter).

Reply:
Done here: https://github.com/mendersoftware/mender-artifact/pull/291/commits/50ea75a6cd6e9e3d30311b24bfee258f32bc9209